R/scoring.R

```diff
@@ -68,77 +62,29 @@ evaluate_predictions <- function(
   assert_that("actual" %in% names(truth_data),
     msg = paste("`truth_data` must contain a column named `actual`")
   )
-  predictions_cards <- left_join(predictions_cards, truth_data,
-    by = intersect(colnames(predictions_cards), colnames(truth_data))
-  )
-  if (is.null(err_measures) || length(err_measures) == 0) {
-    score_card <- predictions_cards
-  } else {
-    err_calls <- lapply(err_measures, function(err_measure) {
-      quo(err_measure(.data$quantile, .data$value, .data$actual))
-    })
-    score_card <- predictions_cards %>%
-      group_by(across(all_of(grp_vars))) %>%
-      summarize(!!!err_calls, .groups = "drop") %>%
-      inner_join(predictions_cards, by = grp_vars, multiple = "all")
-  }
+
+  grp_vars <- c("geo_value", "forecast_date", "target_end_date")
+  err_calls <- lapply(err_measures, function(err_measure) {
+    quo(err_measure(.data$quantile, .data$value, .data$actual))
+  })
+  predictions_cards <- left_join(predictions_cards, truth_data)
+
+  score_card <- predictions_cards %>%
```

Comment:
I guess this is pulling `collapse_cards` inline? Or at least parts of it that aren't conveniences.  Do we have correctness guarantees? I would've expected the pivot_wider to be necessary

Reply:
no tests in this repo, though i suppose i could write a few on the main branch and bring them here.

all testing was done in my repl.